service.js

```diff
@@ -26,18 +26,19 @@ module.exports = function initialize (callback) {
     zmqSock.subscribe('hashblock')
     zmqSock.subscribe('hashtx')
 
-    let lastSequence = 0
+    let sequences = {}
```

Comment:
oh,  I wonder if that is the issue.

Reply:
well, I logged messages to console and sequence for block and tx was different